nixos/modules/virtualisation/docker-containers.nix

```diff
@@ -58,7 +58,7 @@ let
 
         log-driver = mkOption {
           type = types.str;
-          default = "none";
+          default = "journald";
```

Comment:
How would this work? `podman` will only write to `journald` or to `stdout` too?

Reply:
`podman` will also write to stderr/stdout. 
I've updated the service definition with:
```
StandardOutput = "null";
StandardError = "null";
```

We'll leave logging entirely to `podman`.

Reply:
Note: I'd kept the default "none" but podman doesn't have that one.